<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyStuInfo</class>
 <widget class="QWidget" name="ModifyStuInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>学号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_sid"/>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_select">
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>280</y>
     <width>215</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton_back">
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>提交</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_6" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>481</width>
     <height>65</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>姓名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_name"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_4" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>年龄：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_age"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_5" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>性别：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_man">
         <property name="text">
          <string>男</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_woman">
         <property name="text">
          <string>女</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_8" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>361</width>
     <height>65</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>籍贯：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_7" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QComboBox" name="comboBox_province">
         <item>
          <property name="text">
           <string>安徽省</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>北京市</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>广东省</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>广西壮族自治区</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>江苏省</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>上海市</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>四川省</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_city">
         <item>
          <property name="text">
           <string>合肥市</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>滁州市</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>南京市</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>无锡市</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>常州市</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>苏州市</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_county">
         <item>
          <property name="text">
           <string>蜀山区</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>瑶海区</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>江宁区</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_9" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>361</width>
     <height>65</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>专业：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_10" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QComboBox" name="comboBox_college">
         <item>
          <property name="text">
           <string>马克思学院</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>管理学院</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>计算机与软件工程学院</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>理学院</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>外国语学院</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_major"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_class">
         <item>
          <property name="text">
           <string>1班</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2班</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3班</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4班</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5班</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEdit_showPlace">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_showMajor">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
